src/actions/on-gratitude.ts

```diff
@@ -10,11 +11,13 @@ export const onGratitude = async (bot: Bot, message: Message) => {
   const points = await bot.giveGratitudePoints(message.userId, userMentionedId, message.gratitudePoints);
 
   if (points > 0) {
-    const messageToUserMentioned = `¡*<@${message.userId}>* te ha dado *${points}* punto/s de gratitud!`;
+    const messageToUserMentioned = `*<@${message.userId}>* te ha dado *${points}* punto/s de gratitud!`;
     bot.writeMessageToUser(userMentionedId, messageToUserMentioned);
 
     const messageToUserThatMention = `¡Has dado *${points}* puntos a *<@${userMentionedId}>*!`;
     bot.writeMessageToUser(message.userId, messageToUserThatMention);
+
+    Logger.onGratitude();
   } else {
     const messageToUserThatMention = `¡Chacho relajate con los puntitos, que ya no te quedan! ${Emojis.MaikDontAprove}`;
```

Comment:
Not now, but at the future we should move this plain text to a properties files, and inject the value to the template with some value replacement. Right now If we want to change a message we will have to navigate into the code

Reply:
Yes and also for translations, feature for the future 💁‍♂️

Reply:
Yes! 😈